src/e2e/java/teammates/e2e/cases/StudentProfilePageE2ETest.java

```diff
@@ -42,6 +42,27 @@ public void testAll() {
         profilePage.editProfilePhoto();
         profilePage.verifyPhotoSize("220px", "220px");
 
+        ______TS("Typical case: Profile picture ratios");
+        profilePage.fillProfilePic("src/test/resources/images/profile_pic_too_wide.jpg");
+        profilePage.uploadPicture();
+        profilePage.verifyStatusMessage("Your profile picture has been saved successfully");
+
+        profilePage.showPictureEditor(); // In order to upload
+        profilePage.waitForUploadEditModalVisible();
+
+        profilePage.editProfilePhoto();
```

Comment:
This shouldn't be necessary as this block's intention is not to test the edit action

Reply:
Your're right, will remove this

Reply:
After some interesting experience, I've found out that (for whatever reason) `profilePage.showPictureEditor();` is necessary for the tests to succeed. I have no idea why, I can in real time see that after selenium uploads the profile pictures, they are scaled right but for whatever reason the tests still think they are 220px by 220px. The other two lines are however as you said useless